jobs/config.json

```diff
@@ -3300,5 +3300,21 @@
     "--env-file=platforms/gke.env",
     "--env-file=jobs/ci-kubernetes-e2e-gke-latest-upgrade-cluster.env"
   ]
+},
+
+"ci-kubernetes-e2e-gce-gci-latest-upgrade-etcd": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-gci-latest-upgrade-etcd.env"
+  ]
+},
+
+"ci-kubernetes-e2e-gce-gci-1.5-upgrade-etcd": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-gci-1.5-upgrade-etcd.env"
```

Comment:
Thanks.  Also, now I can read the test failure.  Most of it's just noise.  The failure message is right at the end.